src/main/groovy/com/mx/coppuccino/CopyCopConfig.groovy

```diff
@@ -48,6 +50,11 @@ class CopyCopConfig {
   private void copyConfigFile(String resourcePath, String dest) {
     File target = new File(Paths.get(projectRoot, dest).toString())
     ensureDirectory(target.getParentFile())
+
+    if (!overwriteConfigs && target.exists()) {
+      return
+    }
+
```

Comment:
Should we add a command-line arg that allows this to be overwritten (e.g. refresh to the current config files). We could do that as a follow-up, just bringing up the discussion here.

Reply:
I agree. I was able to get that working in part but would need more time than I have to wrap it up. So I do see this as a great future improvement.